requirements.txt

```diff
@@ -45,6 +45,7 @@ deprecation==2.1.0
 detecta==0.0.5; python_version >= "3.6"
 dnspython==2.2.1; python_version >= "3.6" and python_version < "4.0"
 ecos==2.0.10; python_full_version >= "3.6.1" and python_full_version < "4.0.0" and python_version >= "3.7"
+eod==0.2.0; python_version >= "3.8"
```

Comment:
This requirements.txt are actually generated by poetry so this would be wiped next time we submit a pr. Run the following commands to add it:

- `poetry add eod`
- `poetry export  -o requirements.txt --without-hashes`
- `poetry export  -o requirements-full.txt --extras prediction --without-hashes --dev`